app/models/sales_log.rb

```diff
@@ -350,4 +350,8 @@ def mortgage_deposit_and_grant_total
   def beds_for_la_sale_range
     beds.nil? ? nil : [beds, LaSaleRange::MAX_BEDS].min
   end
+
+  def combined_income
+    income1 + income2
```

Comment:
what if one of them is nil?

Reply:
this is only for displaying the combined income in the soft validation interruption screen, and we only run the combined validation when both incomes are present

Reply:
I'd rather not but I'll put an explicit nil check

Reply:
I agree it seems sensible to catch potential errors in case someone else ends up using this in future without thinking too hard.

sorry about the half written comment earlier, I started editing it then forgot, the above is the only important bit

Reply:
no problemo